content/docs/portals.md

```diff
@@ -4,21 +4,22 @@ title: Portals
 permalink: docs/portals.html
 ---
 
-Portals provide a first-class way to render children into a DOM node that exists outside the DOM hierarchy of the parent component.
+Los portales proporcionan un camino de primera clase para mostrar los hijos en un nodo del DOM, que existe afuera de la jerarquía del DOM del componente padre.
 
 ```js
 ReactDOM.createPortal(child, container)
 ```
 
-The first argument (`child`) is any [renderable React child](/docs/react-component.html#render), such as an element, string, or fragment. The second argument (`container`) is a DOM element.
+El primer argumento (`child`) es cualquier [hijo renderizable React](/docs/react-component.html#render), como un elemento, cadena de caracteres o fragmento. El segundo argumento (`container`) es un elemento DOM.
 
-## Usage
+## Uso
 
-Normally, when you return an element from a component's render method, it's mounted into the DOM as a child of the nearest parent node:
+Normalmente, cuando devuelve un elemento del método de render de un componente, este se monta en el DOM como un elemento hijo del nodo padre más cercano:
```

Comment:
May we use "retornar" instead of "devolver"? It creates a more direct relation with the `return` statement. 

Also, for this scenario, the word should end with an "s": "retorna*s*" / "devuelve*s*"